### PR TITLE
Extend docs on customising object display

### PR DESCRIPTION
**Pull request by takluyver**
I just answered a question on #10889 with information that I should be able to point to in the docs, but I couldn't find it anywhere. So here's a bit more information on customising how objects are displayed.

### CODE REVIEW

File: docs/source/config/integrating.rst [hunk 1]
```diff
@@ -40,6 +40,38 @@ For example::
...
+
+    def foo_html(obj):
+        return '<marquee>Foo object %s</marquee>' % obj.name
+
+    html_formatter = get_ipython().display_formatter.formatters['text/html']
+    html_formatter.for_type(Foo, foo_html)
+
+    # Or register a type without importing it - this does the same as above:
+    html_formatter.for_type_by_name('bar.Foo', foo_html)
+
```
Comment (Carreau) [inline docs/source/config/integrating.rst]: I think these are also documented in the `display()` docstring. Should it be cross linked there ?

**Review by Carreau — APPROVED** [review]
+1, feel free to merge if you want to add the crosslink to display later.

File: docs/source/config/integrating.rst (hunk 1 above)
Comment (takluyver) [reply docs/source/config/integrating.rst]: Done.

Comment (Carreau) [reply docs/source/config/integrating.rst]: Sweet, last read and merge.

**Pull request merged**